frida-gum/src/range_details.rs

```diff
@@ -0,0 +1,60 @@
+use frida_gum_sys as gum_sys;
+use std::ffi::CStr;
+use std::os::raw::c_void;
+
+use crate::{NativePointer, MemoryRange};
+
+pub struct FileMapping {
+    file_mapping: gum_sys::GumFileMapping,
+}
+
+impl FileMapping {
+    pub fn new_from_raw(file: gum_sys::GumFileMapping) -> Self {
+        Self {
+            file_mapping: file
+        }
+    }
+
+    pub fn get_path(&self) -> &str {
```

Comment:
Instead of using `get_` we just use the name for getters. E.g.: `path`. Ditto for `offset` and `size`.

Reply:
Got it.